README.md

```diff
@@ -1,2 +1,4 @@
 # wonderland-8thwall-tracking
-Components of the 8thwall AR tracking integration.
```

Comment:
Some basic description might still be useful, a `Setup`, `Examples` (Short description per example) and maybe a License section.

We probably want to split off the general AR framework into its own repo (or make it part of the default components?) and then add a reference with a short code snippet on how to register the AR provider.